etc/env/example.env

```diff
@@ -314,3 +321,16 @@ CANCOGEN_FEDERATION_URL=http://federation-service:4232/federation/search
 CANDIG_DATA_PORTAL_VERSION=v0.1.0
 CANDIG_DATA_PORTAL_PORT=2543
 CANDIG_DATA_PORTAL_URL=http://${CANDIG_DOMAIN}:${CANDIG_DATA_PORT}/data-portal
+
+# graphql-interface
+GRAPHQL_CONDA_SOURCE=condaforge
```

Comment:
why does the container require conda?

Reply:
Fixed: See https://github.com/CanDIG/CanDIGv2/pull/118